plugin/build.gradle.kts

```diff
@@ -20,16 +21,19 @@ repositories {
 
 dependencies {
     compileOnly(gradleApi())
-    compileOnly(kotlin("gradle-plugin", "1.2.21"))
-    compileOnly("com.android.tools.build:gradle:3.0.0")
-    compileOnly("org.jetbrains.kotlin:kotlin-native-gradle-plugin:0.6")
-    compile("net.swiftzer.semver:semver:1.0.0")
+    implementation(kotlin("gradle-plugin", "1.2.21"))
```

Comment:
Can you change back to `compileOnly`? This 3 dependencies should be optional, so users of this plugin will not get them as a transitive dependency.

Reply:
Using `compileOnly` here is unfortunately not supported by TestKit automatic plugin classpath injection into tests. It can be done with some additional gymnastic, I'll look into it but it will make the testing setup more complicated.

Reply:
We don't want to add more dependencies than are strictly necessary to run the plugin.